src/Microsoft.Health.Fhir.CosmosDb/Features/Storage/StoredProcedures/Upsert/upsertWithHistory.js

```diff
@@ -49,7 +41,7 @@ function upsertWithHistory(doc, matchVersionId, allowCreate, keepHistory) {
         let isAccepted = collection.createDocument(collectionLink, doc, { disableAutomaticIdGeneration: true }, callback);
 
         if (!isAccepted) {
-            throw new Error(errorMessages.RequestNotQueued);
+            throw createRequestNotQueuedError();
```

Comment:
Since we are touching the code, can we also fix this as well to be consistent? This one throws 503 but the other one throws 413 entity too large. I think we should standardize it to 429 too many requests. What do you think?

Reply:
Created #451 to address this.